packages/inter-protocol/test/smartWallet/test-oracle-integration.js

```diff
@@ -308,3 +316,207 @@ test.serial('errors', async t => {
+    // @ts-expect-error TS can't tell that it's going to satisfy the @returns.
+    E(E(zoe).getInvitationIssuer())
+      .getBrand()
+      .then(brand => {
+        /** @type {Amount<'set'>} */
+        const invitationsAmount = NonNullish(balances.get(brand));
+        console.log('DEBUG invitationsAmount', invitationsAmount);
+        t.is(invitationsAmount?.value.length, len);
+        return invitationsAmount.value.filter(i => i.description === desc);
+      });
+
+  console.log('DEBUG computedState.balances', computedState.balances);
+
+  const proposeInvitationDetails = await getInvitationFor(
+    EC_INVITATION_MAKERS_DESC,
+    2,
+    computedState.balances,
+  );
+
+  t.is(proposeInvitationDetails[0].description, EC_INVITATION_MAKERS_DESC);
+  t.is(proposeInvitationDetails[0].instance, econCharter, 'econCharter');
+  t.is(
+    // @ts-expect-error cast amount kind
+    currentPurseBalance(await headValue(currentSub), invitationBrand).length,
+    2,
+    'two invitations deposited',
+  );
+
+  // The purse has the invitation to get the makers ///////////
+
+  /** @type {import('@agoric/smart-wallet/src/invitations').PurseInvitationSpec} */
+  const getInvMakersSpec = {
+    source: 'purse',
+    instance: econCharter,
+    description: EC_INVITATION_MAKERS_DESC,
+  };
+
+  /** @type {import('@agoric/smart-wallet/src/offers').OfferSpec} */
+  const invMakersOffer = {
+    id: 44,
+    invitationSpec: getInvMakersSpec,
+    proposal: {},
+  };
+
+  await offersFacet.executeOffer(invMakersOffer);
+
+  /** @type {import('@agoric/smart-wallet/src/smartWallet.js').CurrentWalletRecord} */
+  let currentState = await headValue(currentSub);
+  t.is(
+    // @ts-expect-error cast amount kind
+    currentPurseBalance(currentState, invitationBrand).length,
+    1,
+    'one invitation consumed, one left',
+  );
+  t.deepEqual(Object.keys(currentState.offerToUsedInvitation), ['44']);
+  t.is(
+    currentState.offerToUsedInvitation[44].value[0].description,
+    'charter member invitation',
+  );
+
+  console.log('DEBUG purses', util.inspect(currentState.purses, false, 8));
+  console.log(
+    'DEBUG offerToUsedInvitation',
+    util.inspect(currentState.offerToUsedInvitation, false, 8),
+  );
+  // Call for a vote ////////////////////////////////
+
+  const feed = await E(agoricNames).lookup('instance', 'ATOM-USD price feed');
+  t.assert(feed);
+
+  /** @type {import('@agoric/smart-wallet/src/invitations').ContinuingInvitationSpec} */
+  const proposeInvitationSpec = {
+    source: 'continuing',
+    previousOffer: 44,
+    invitationMakerName: 'VoteOnApiCall',
+    invitationArgs: harden([feed, 'addOracle', [newOracle], 2n]),
+  };
+
+  /** @type {import('@agoric/smart-wallet/src/offers').OfferSpec} */
+  const proposalOfferSpec = {
+    id: 45,
+    invitationSpec: proposeInvitationSpec,
+    proposal: {},
+  };
+
+  await offersFacet.executeOffer(proposalOfferSpec);
+  await eventLoopIteration();
+
+  // vote /////////////////////////
```

Comment:
Consider using `t.log` in place of most comments of this sort...

```suggestion
  t.log('now the EC member(s) vote')
```